js/components/changesetNotice.js

```diff
@@ -191,6 +192,7 @@ const create_unified_changeset_notice = ({
     let cancel_button = {
         text: cancel_text,
         addClass: 'btn-sm',
+        promptTrigger: true,
```

Comment:
you've tested this on our text-entry fields and it doesn't result in cancelling when you press enter?  That was my only worry in adding promptTrigger: true on non-modal popups

Reply:
It does but I figured that was better than activating the ok button.  Were you hoping for a more neutral component being activated and the user needing to specifically select Ok or Cancel?

Reply:
I'm not sure, I guess this is more a UX question, but my instinct for the most neutral change would be to add `promptTrigger` for non keyboard-input pNotifies, and manually direct focus to the text box in case of input-required ones?  That might take a lot more overhaul.  Maybe this is one to direct to @jennyhavah?

Reply:
Oh interesting, I can play around with it a bit and see if it's easy.